File: CS/GradientHeightMap/Model/ImageDataModel.cs
using System.Windows.Media;

namespace GradientHeightMap {
    public class ImageData {
        public DoubleCollection XArguments { get; set; }
        public DoubleCollection YArguments { get; set; }
        public DoubleCollection Values { get; set; }
        public ImageData(DoubleCollection xArguments, DoubleCollection yArguments, DoubleCollection values) {
            this.XArguments = xArguments;
            this.YArguments = yArguments;
            this.Values = values;
        }
    }
}

File: VB/GradientHeightMap/Model/ImageDataModel.vb
Imports System.Windows.Media

Namespace GradientHeightMap
	Public Class ImageData
		Public Property XArguments() As DoubleCollection
		Public Property YArguments() As DoubleCollection
		Public Property Values() As DoubleCollection
		Public Sub New(ByVal xArguments As DoubleCollection, ByVal yArguments As DoubleCollection, ByVal values As DoubleCollection)
			Me.XArguments = xArguments
			Me.YArguments = yArguments
			Me.Values = values
		End Sub
	End Class
End Namespace